slam-ws/src/caffeine/urdf/caffeine.urdf.xacro

```diff
@@ -306,6 +306,49 @@
       <origin xyz="${base_width / 2.0} 0 ${base_height / 2.0 + hokuyo_height / 2.0 + 0.5}" rpy="0 0 ${-135 * pi / 180}"/>
    </joint>
 
+   <!-- ZED Camera -->
+  <link name="zed_camera_link">
+      <visual>
+         <origin xyz="0 0 0" rpy="0 0 0"/>
+            <!-- TODO: should be using the mesh but couldn't get it to work -->
+         <geometry>
+            <box size="${ZED_camera_length} ${ZED_camera_width} ${ZED_camera_height}"/>
+         </geometry>
+       </visual>  
+       <collision>
+           <origin xyz="0 0 0" rpy="0 0 0"/>
+           <geometry>
+               <box size="${ZED_camera_length} ${ZED_camera_width} ${ZED_camera_height}"/>
+           </geometry>
+       </collision>
+       <inertial>
+           <mass value="${ZED_camera_mass}"/> <!-- Obtained from google search -->
```

Comment:
Unnecessary comment. If you want this comment, put it in the constants file since it's value is what you obtained from Google

Reply:
Deleted